File: March'22 - May'22/Week 02/Assignment/SimpleDivision.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int testCases = 1;
    cin >> testCases;
    while (testCases--)
    {
        int n, x, y, i;
        cin >> n >> x >> y;
        int arr[n];
        int count = 0;
        for (int i = 0; i < n; i++)
        {
            cin >> arr[i];
            if ((arr[i] <= x) && (arr[i] % y == 0))
                count++;
        }
        cout << count << endl;
    }
    return 0;
}

File: March'22 - May'22/Week 02/Assignment/SimpleDivision.py
def get_array():
    return list(map(int, input().split()))


numberOfTestCases = int(input())
for i in range(numberOfTestCases):
    count = 0
    arrayOne = get_array()
    arr = get_array()
    for j in range(0, arrayOne[0]):
        if(arr[j] % arrayOne[2] == 0 and arr[j] <= arrayOne[1]):
            count += 1
    print(count)
